de.fu_berlin.inf.dpp/src/de/fu_berlin/inf/dpp/EclipsePluginLifecycle.java

```diff
@@ -55,8 +55,10 @@ private EclipsePluginLifecycle(Saros saros) {
 
   @Override
   protected void initializeContext(final ContainerContext containerContext) {
-    FeedbackPreferences.setPreferences(
-        containerContext.getComponent(org.osgi.service.prefs.Preferences.class));
+    if (Boolean.getBoolean("de.fu_berlin.inf.dpp.feedback.ENABLED")) {
```

Comment:
This call is now repeated in multiple files using a hardcoded string. What about abusing putting this into a central place ?

Reply:
I was not sure whether there is a common place for property checks. Otherwise I would add a corresponding static method to a feedback class